opendbc/car/subaru/interface.py

```diff
@@ -42,6 +42,11 @@ def _get_params(ret: structs.CarParams, candidate: CAR, fingerprint, car_fw, alp
     else:
       CarInterfaceBase.configure_torque_tune(candidate, ret.lateralTuning)
 
+    if ret.flags & SubaruFlags.LKAS_ANGLE:
+      ret.steerActuatorDelay = 0.3
+      ret.safetyConfigs[0].safetyParam |= SubaruSafetyFlags.LKAS_ANGLE.value
+      ret.dashcamOnly = is_release
```

Comment:
@adeebshihadeh is this what you meant by
> let's move this to the safety mode

comment from [here](https://github.com/commaai/opendbc/pull/2864#discussion_r2607768704)